tests/unit_tests/test_hybrid_search_return_fields.py

```
@@ -0,0 +1,208 @@
+        return BaseClient._build_search_parm(  # type: ignore[misc]
+            self,
+            query=query,
+            knn=knn,
+            rank=rank,
+            n_results=n_results,
+            return_fields=return_fields,
+            dimension=dimension,
+            **kwargs,
+        )
+
+    def _execute_query_with_cursor(self, conn, sql, params, use_context_manager=True):  # type: ignore[no-untyped-def]
```

Ruff (0.14.13)

Unused method argument: `conn`

(ARG002)